RELEASE_NOTES.md

```diff
@@ -1,3 +1,53 @@
+## Cookstyle 8.0
+
+### RuboCop 1.75.2
+
+The RuboCop engine that powers Cookstyle has been updated to 1.75.2 with a HUGE number of under the hood improvements and autocorrection bug fixes.
+
+### Updated default Ruby release
+
+Cookstyle now defaults to Ruby 2.6, enabling new language features, while still helping users on legacy Chef Infra Client releases to upgrade their cookbooks.
```

Comment:
3.0?

Reply:
Cookstyle previously defaulted to 2.5, which helped with really old Infra Client upgrades. You want to upgrade your cookbooks before doing the client release. By targeting an old version of Ruby by default you can resolve deprecations in your cookbooks without any modern Ruby-isms sneaking it. If you want to target the latest Infra Client release then you can bump up the target infra client and ruby versions in a .rubocop.yml file.

I did bump Chefstyle to 3.0 though since that's really just for internal use and 2.x support in Chef projects is all gone now.